### PR TITLE
Fix broken WMS legend image for labelled layers

### LINKED ISSUE #209: Legend for labelled WMS layers not shown
When activating labels for a WMS layer via context menu and then changing the style the legend image breaks:

grafik

### PR DESCRIPTION
**Pull request by chrismayer**
Fixes the broken WMS legend image for labelled layers.

Closes #209

### CODE REVIEW

File: app/util/Legend.js [hunk 1]
```diff
@@ -119,6 +124,20 @@ Ext.define('CpsiMapview.util.Legend', {
...
+     * @param  {String} layers WMS LAYERS parameter value, e.g. (layer1,layer2)
+     * @return {String}        WMS LAYERS parameter without duplicates
+     */
+    getUniqueLayersParam: function (layers) {
+        var layerList = Ext.isArray(layers) ? layers : layers.split(',');
+        // remove any duplicate layer names
+        var reducedLayerList = Ext.Array.unique(layerList);
+
+        return reducedLayerList.join(',');
+    },
```
Comment (geographika) [inline app/util/Legend.js]: Extra comma here should be removed?

Comment (chrismayer) [reply app/util/Legend.js]: I don't think so. `join(',')` handles this and avoids a trailing comma. See [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join). Or do I oversee something?

**Comment by chrismayer** [comment]
Thanks for the review @JakobMiksch! Waiting for @geographika 's go (because of the open question) and the I am going to merge this.

File: app/util/Legend.js (hunk 1 above)
Comment (geographika) [reply app/util/Legend.js]: Sorry I meant line 141. This used to break things on IE7 :)

Comment (chrismayer) [reply app/util/Legend.js]: Oh now I see ... yes the good old IE 7 times :laughing: I guess the JS builder takes care of this, but to be save I'll remove this.

**Comment by chrismayer** [comment]
What is failing here is the Azure pipeline, not the unit tests. I assume the Azure pipeline should not be executed here at all, correct @geographika?.

**Comment by geographika** [comment]
@chrismayer - the pipeline is used to build a minified version. It was working fin eon my branch before merging. Please ignore while I update it and go ahead with merging this.

**Comment by chrismayer** [comment]
Thanks for clarification @geographika. I'll merge now.

**Pull request merged**